src/pages/not-found/NotFound.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Button } from "@material-ui/core";
+import Box from "@material-ui/core/Box";
+import { ErrorOutline } from "@material-ui/icons";
+import { Link } from "react-router-dom";
+
+const NotFound = () => (
+  <Box
+    textAlign="center"
+    position="absolute"
+    top="50%"
+    left="50%"
+    style={{ transform: "translate(-50%, -50%)" }}
+  >
```

Comment:
Would it be cleaner to use use/makeStyles here? the answer can again be NO 😅

Reply:
it would 😄 fixed!